service/src/main/java/org/apache/griffin/core/integration/JobEventHook.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.griffin.core.integration;
+
+import org.apache.griffin.core.exception.GriffinException;
+
+public class JobEventHook implements GriffinHook {
+    @Override
+    public void onEvent(GriffinEvent event) throws GriffinException {
+        throw new GriffinException.UnImplementedException(
+            "This method needs to be reimplemented by event-consuming purpose");
```

Comment:
I think griffin will implement and fire to hooks here, right?

Reply:
if we think about exporting interface and hope user to implement method details. we should let it empty or throw a exception tip here

Reply:
By the way, if this was for testing purposes, we could have test hook defined in test code itself (like in case of hive metadata service tests).